CONTRIBUTING.md

```diff
@@ -0,0 +1,36 @@
+# Contribution Guidelines
+We welcome feedback, bug reports, and pull requests!
+For pull requests (PRs), please stick to the following guidelines:
+* Before submitting a PR, verify that [an issue](https://github.com/luno/REPONAME/issues) exists that describes the bug fix or feature you want to contribute. If there's no issue yet, please [create one](https://github.com/luno/REPONAME/issues/new/choose).
+* Fork REPONAME on your GitHub user account, make code changes there, and then create a PR against the main REPONAME repository.
```

Comment:
Is REPONAME a keyword that will be replaced when this default is shown on a repo? If not, is there maybe such a thing that anyone knows about? It's a bit meh seeing `REPONAME`

Reply:
Doesn't really look like it, maybe we can write this a little better like 'Fork the repo' and 'create a PR against the main repository'?

Reply:
Yes I don't think this is possible unfortunately, might just have to say "the repo"

Reply:
Chat gippity said we should use a github action 😂

Reply:
Yeah no, I agree, it just seemed like an amusing amount of unneeded effort